private-templates-service/client/src/components/Gallery/Gallery.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from "react";
+import AdaptiveCard from "../AdaptiveCard";
+import { Container } from "./styled";
+
+class Gallery extends React.Component {
+	render() {
+		return (
+			<Container>
+				<AdaptiveCard />
```

Comment:
This 3 x <.AdaptiveCard /.> is just a temp thing for now right?

Reply:
Yes, I will "un-hardcode" it once I start grabbing adaptive card templates from the database (via the backend).